src/app/services/office.service.ts

```diff
@@ -751,4 +774,102 @@ export class OfficeService {
       });
     });
   }
+
+  private hideSdt = (node: Node) => {
```

Comment:
Bitte kommentieren. Ich habe keine Ahnung was für Elemente hier versteckt werden.

Reply:
Die Elemente sind Tags in OOXML. Ich kann ja Links auf die Doku hinzufügen. Aber wer den Code verstehen will, muss sich ein bißchen in OOXML einlesen.

Reply:
Klar wird man sich in die Doku einlesen müssen. Aber ich möchte das nicht jedes mal machen müssen, wenn ich wieder vergessen habe für was `Sdt` steht. Deswegen wäre ein Kommentar wie `Versteckt Bildknoten` mit einem Link zur OOXML Doku durchaus hilfreich.